lib/cache.js

```diff
@@ -62,9 +72,14 @@ function tryLoad(pages, index, done) {
 }
 
 function isPage(file) {
-  return file.match(/pages\/.*\.md$/);
+  return file.match(/.*\.md$/);
 }
 
+function onPlatform(os) {
+  return function(file) {
+    return _.startsWith(file, 'common') || _.startsWith(file, os);
```

Comment:
Tssk including the whole of lodash for this :P? 

`file.match(/^common/)` for the first bit, second you can build from string.

I really don't care though, lodash is great.